### PR TITLE
[12.0][UPD] l10n_fr_fec: update fr.po

### PR DESCRIPTION
**Pull request by sbidoul**
This is obtained with msgmerge -U.

Apparently this file escaped the conversion of translation files for v12?

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_fr_fec/i18n/fr.po [hunk 1]
```diff
@@ -17,111 +17,289 @@ msgstr ""
...
+#: model_terms:ir.ui.view,arch_db:l10n_fr_fec.account_fr_fec_view
+msgid "ValidDate"
+msgstr ""
+
+#. module: l10n_fr_fec
+#: model_terms:ir.ui.view,arch_db:l10n_fr_fec.account_fr_fec_view
+msgid "We use partner.id"
+msgstr ""
+
+#~ msgid "Error:"
```
Comment (mart-e) [inline addons/l10n_fr_fec/i18n/fr.po]: can you remove theses ones?

File: addons/l10n_fr_fec/i18n/fr.po [hunk 2]
```diff
@@ -17,111 +17,289 @@ msgstr ""
...
-#: field:account.fr.fec,export_type:0
-msgid "Export Type"
-msgstr "Type d'export"
+#: model_terms:ir.ui.view,arch_db:l10n_fr_fec.account_fr_fec_view
+msgid "Technical Name"
+msgstr ""
+
+#. module: l10n_fr_fec
+#: model_terms:ir.ui.view,arch_db:l10n_fr_fec.account_fr_fec_view
+#, fuzzy
```
Comment (mart-e) [inline addons/l10n_fr_fec/i18n/fr.po]: a fuzzy one, the match seems bad as the translation seems incomplete

**Comment by sbidoul** [comment]
@mart-e done

**Comment by mart-e** [comment]
Thanks for the update !

robodoo r+

**Pull request closed without merging**